controller/workitem.go

```diff
@@ -565,6 +565,36 @@ func workItemIncludeHasChildren(appl application.Application, ctx context.Contex
 	}
 }
 
+// includeParentWorkItem adds the parent of given WI to relationships & included object
+func includeParentWorkItem(ctx context.Context, appl application.Application) WorkItemConvertFunc {
+	return func(request *http.Request, wi *workitem.WorkItem, wi2 *app.WorkItem) {
+		var parentID *uuid.UUID
+		var err error
+		repo := appl.WorkItemLinks()
+		if repo != nil {
+			parentID, err = appl.WorkItemLinks().GetParentID(ctx, wi.ID)
```

Comment:
Why do you use `appl.WorkItemLinks()` here when you have created `repo` two lines above?

Reply:
my bad, thanks. Done in https://github.com/fabric8-services/fabric8-wit/pull/1592/commits/d64aa3a7bd28e06ea6a169926ea1e1f3b09216b0